<a href="https://colab.research.google.com/github/joseydyrueda-code/programacion1/blob/main/Quiz26marzo.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Joseydy Andrea Perozo Perozo Rueda

*   Matematicas
*   semestre 1
*   2260665






In [6]:
%%writefile pedirnotas.py
def pedir_nombre():
  return input("ingrese el nombre del estudiante: ")

def pedir_notas():
  n1 = float(input("ingrese la nota 1: "))
  n2 = float(input("ingrese la nota 2: "))
  n3 = float(input("ingrese la nota 3: "))
  n4 = float(input("ingrese la nota 4: "))
  return n1, n2, n3, n4

Writing pedirnotas.py


In [9]:
import pedirnotas
nombre = pedirnotas.pedir_nombre()



ingrese el nombre del estudiante: joss


In [10]:
import importlib
import pedirnotas
importlib.reload(pedirnotas)
notas = pedirnotas.pedir_notas()


ingrese la nota 1: 2
ingrese la nota 2: 4
ingrese la nota 3: 4
ingrese la nota 4: 5


In [12]:
%%writefile promedio.py
import statistics
def calcular_promedio(n1, n2, n3, n4):
   return(n1+n2+n3+n4) / 4

def mediana(n1, n2, n3, n4):
  return statistics.median([n1, n2, n3, n4])

def moda(n1, n2, n3, n4):
  return statistics.mode([n1, n2, n3, n4])


Overwriting promedio.py


In [14]:
import promedio
#import importlib
importlib.reload(promedio)
prom = promedio.calcular_promedio(notas[0], notas[1], notas[2], notas[3])
mediana = promedio.mediana(notas[0], notas[1], notas[2], notas[3])
moda = promedio.moda(notas[0], notas[1], notas[2], notas[3])

print("estudiante", nombre)
print("promedio", prom)
print("mediana", mediana)
print("moda", moda)

if prom >= 3:
  print("aprobado")

else:
  print("reprobado")
  #falta definir el pomedio


estudiante joss
promedio 3.75
mediana 4.0
moda 4.0
aprobado



#Análisis de datos:




*   Media Aritmética: En promedio, el estudiante obtuvo 3.75
*   Mediana: El valor que divide el 50% de las calificaciones es 4.0
*   La calificacion mas repetida es 4.0

*   El desempeño del estudiante es bueno, porque obtuvo un
 promedio cercano a 3.8 y cuenta con minimo el 50% de sus notas en 4.0




